ladybug_display_schema/visualization.py

```diff
@@ -293,27 +269,76 @@ class GraphicContainer(NoExtraBaseModel):
         'If None, the default units of the data_type will be used.'
     )
 
+
+class VisualizationSet(NoExtraBaseModel):
+    """A visualization where multiple data streams correspond to the same geometry."""
+
+    type: constr(regex='^VisualizationSet$') = 'VisualizationSet'
+
+    geometry: Union[Mesh2D, Mesh3D, Polyface3D, List[GEOMETRY_UNION]] = Field(
+        ...,
+        description='A ladybug-geometry object or list of ladybug-geometry objects that '
+        'is aligned with the values in the input data_sets. If a Mesh or Polyface '
+        'is specified here, it is expected that the number of values match the '
+        'number of faces or the number of vertices. If a list of geometry objects '
+        'is specified (ie. a list of Point3Ds), it is expected that the length '
+        'of this list align with the number of values.'
+    )
+
+    data_sets: List[VisualizationData] = Field(
+        ...,
+        min_items=1,
+        description='An list of VisualizationData objects representing the data sets '
+        'that are associated with the input geometry.'
+    )
+
+    default_data: int = Field(
+        0,
+        description='An integer to denote which of the input data_sets should be '
+        'displayed by default.'
+    )
+
+    context: List[GEOMETRY_UNION] = Field(
+        None,
+        description='An optional list of ladybug-geometry objects that should be '
+        'displayed alongside the input geometry to help give the geometry context. '
+        'Typically, these will display in wireframe around the geometry.'
```

Comment:
This is an interesting comment here and opens up a different conversation about display properties. We talked about this before briefly but now might be a good time to bring it up again. I will write a longer post on Basecamp to discuss this more.

Reply:
Already discussed on Basecamp and it's been implemented!